tests/integration/test_schema.py

```diff
@@ -1074,21 +1074,23 @@ async def test_schema_versions_deleting(registry_async_client: Client, trail: st
         "type": "record",
         "name": schema_name,
         "fields": [{
-            "name": "f1",
-            "type": "string",
```

Comment:
This is an *extreme* nitpick. But maybe we should agree on a default here.

I always add the trailing comma for two reasons:

- YAPF formatting is better in this case
- When a new entry is added later, we don't have to modify the previous line, it makes using `git blame` easier because only the relevant line is changed. This is especially useful for function arguments

Because of the above I always add trailing commas. We should probably figure something out so that we don't have every other commit commas being added and then removed

Reply:
Yes, it seems to be a widely adopted practise to add the trailing comma in Python arrays and I completely understand why that practise is there. It just that many languages don't support that and at least my muscle memory doesn't write it automatically.

But yes, we can agree of having the trailing comma. I'm all for it.